package.json

```diff
@@ -35,7 +35,7 @@
     "build": "npm run build-esb && npm run build-tsc",
     "test": "jest",
     "lint": "npx eslint \"src/**/*.@(tsx|ts|jsx|js)\" --fix ; npx prettier --check \"src/**/*.@(tsx|ts|jsx|js)\" --write",
-    "prepare": "npm run build",
+    "prepare": "npm run build && husky install",
```

Comment:
Mind chcecking if it executes after package installation? As far as i remember husky was designed this way for a particular reason that it should not be fired if you `npm i` it. I think it would be better to leave it be and add section in readme explaining how to initialize husky.

Reply:
yup, the script executes on `npm install`:
![CleanShot 2022-03-28 at 10 24 52@2x](https://user-images.githubusercontent.com/4144459/160357408-3f853ed7-8e23-4a64-8a57-af5728b9c09a.png)
![CleanShot 2022-03-28 at 10 25 16@2x](https://user-images.githubusercontent.com/4144459/160357414-165615b6-7c75-4c46-b1c6-429e3d5f5ae2.png)

I think that was the case with husky v4 - it used to be in `postinstall` script which was problematic and now they recommend using `prepare` script. [Article about this change](https://blog.typicode.com/husky-git-hooks-autoinstall/), [Husky documentation](https://typicode.github.io/husky/#/?id=install)

<img width="829" alt="CleanShot 2022-03-28 at 10 29 09@2x" src="https://user-images.githubusercontent.com/4144459/160357969-875cbaea-22ab-42a9-ae9e-a1d2877cfcb8.png">

Reply:
ok, I didn't get the message: We don't want husky to be installed when consumers of macaw are installing it in their projects. I'll remove it from package.json and instead add note to README